src/lib/mina_version/normal/gen.sh

```diff
@@ -1,41 +1,34 @@
-#!/bin/bash
-set -e
+#!/usr/bin/env bash
+set -euo pipefail
 
-commit_id_short=$(git rev-parse --short=8 --verify HEAD)
-CWD=$PWD
-
-if [ -n "$MINA_BRANCH" ]; then
-  branch="$MINA_BRANCH"
-else
-  branch=$(git rev-parse --verify --abbrev-ref HEAD || echo "<none found>")
-fi
+branch="${MINA_BRANCH-$(git rev-parse --verify --abbrev-ref HEAD || echo "<unknown>")}"
```

Comment:
Changes semantics, we should be sure no one is using a check on `"<none found>"`